OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -39,20 +38,8 @@ public virtual ISpriteSequence CreateSequence(ModData modData, TileSet tileSet,
 			}
 
 			foreach (var kvp in nodes)
-			{
 				using (new Support.PerfTimer("new Sequence(\"{0}\")".F(node.Key), 20))
-				{
-					try
-					{
-						sequences.Add(kvp.Key, CreateSequence(modData, tileSet, cache, node.Key, kvp.Key, kvp.Value));
-					}
-					catch (FileNotFoundException ex)
-					{
-						// Eat the FileNotFound exceptions from missing sprites
-						OnMissingSpriteError(ex.Message);
```

Comment:
I have mixed feelings about this.  While we *do* want the game to eagerly fail if sequence artwork is missing, this basically completely ruins the usefulness of the utility check command (which now fails verbosely on the first error instead of listing all).  It would be nice if we could keep both.